.gitignore

```diff
@@ -4,6 +4,9 @@
 # Log file
 *.log
 
+# Env file
+.env
+
```

Comment:
you don`t need to commit only file .env.blank, but .env file should be in project